12-interpolation.md

```diff
@@ -23,8 +23,8 @@ Interpolation can also be nested:
 - `${VARIABLE?$FOO}`
 - `${VARIABLE:-${FOO:-default}}`
 
-Other extended shell-style features, such as `${VARIABLE/foo/bar}`, are not
-supported by the Compose specification.
+Other extended shell-style features, such as `${VARIABLE/foo/bar}`, is not
```

Comment:
Should be conjugated with `features`, no?
```suggestion
Other extended shell-style features, such as `${VARIABLE/foo/bar}`, are not
```

Reply:
Woops! Yes!